target/linux/ath79/dts/qca9557_8dev_rambutan.dts

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	compatible = "8dev,rambutan", "qca,qca9557";
+	model = "8devices Rambutan";
+
+	keys: keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+	};
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x300000>;
+			read-only;
+		};
+
+		partition@300000 {
+			label = "u-boot-env";
+			reg = <0x300000 0x200000>;
+		};
+
+		art: partition@500000 {
+			label = "art";
+			reg = <0x500000 0x100000>;
+			read-only;
+		};
+
+		partition@600000 {
+			label = "ubi";
+			reg = <0x600000 0x7a00000>;
+		};
+	};
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy0: ethernet-phy@0 {
+		reg = <0>;
+		reset-gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+	};
+};
+
+&mdio1 {
+	status = "okay";
+
+	phy1: ethernet-phy@1 {
+		reg = <0>;
+		reset-gpios = <&gpio 23 GPIO_ACTIVE_LOW>;
+		phy-mode = "sgmii";
+		at803x-override-sgmii-link-check;
+	};
+};
+
+&eth0 {
+	status = "okay";
+
+	mtd-mac-address = <&art 0x0>;
+	phy-mode = "mii";
+	phy-handle = <&phy0>;
+};
+
+&eth1 {
+	status = "okay";
+
+	pll-data = <0x17000000 0x101 0x1313>;
+	phy-handle = <&phy1>;
+	phy-mode = "sgmii";
+	qca955x-sgmii-fixup;
+	mtd-mac-address = <&art 0x6>;
+};
+
+&wmac {
+	status = "okay";
+
+	gpio-controller;
+	mtd-cal-data = <&art 0x1000>;
+};
+
+&pcie0 {
+	status = "okay";
+};
+
+&usb_phy0 {
+	status = "okay";
+};
+
+&usb0 {
+	status = "okay";
+
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	hub_port0: port@1 {
+		reg = <1>;
+		#trigger-source-cells = <0>;
+	};
```

Comment:
Those ports are only required if you link them to an USB LED.

Reply:
Fixed.